src/main/java/seedu/agendum/commons/core/Messages.java

```diff
@@ -5,10 +5,11 @@
  */
 public class Messages {
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_UNKNOWN_COMMAND_WITH_SUGGESTION = "Unknown command, did you mean '%1$s'?";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "The task index provided is invalid";
-    public static final String MESSAGE_TASKS_LISTED_OVERVIEW = "%1$d tasks listed!";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "We don't recognise this command";
+    public static final String MESSAGE_UNKNOWN_COMMAND_WITH_SUGGESTION = "Did you mean '%1$s'?";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT =
+            "We don't understand your command format. Try\n%1$s";
+    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "Hey, the task index given is invalid";
```

Comment:
id might be better. IMO easier to think that every "task has an id" than every "task has an index"